scripts/check-tools.sh

```
@@ -1,78 +1,130 @@
-  command -v "$cmd_name" >/dev/null 2>&1 || {
-    echo "Error: '$cmd_name' is not installed." >&2
-    exit 1
-  }
-  return 0
+# Visuals
+BOLD="$(tput bold 2>/dev/null || echo '')"
+RESET="$(tput sgr0 2>/dev/null || echo '')"
+GREEN="$(tput setaf 2 2>/dev/null || echo '')"
+YELLOW="$(tput setaf 3 2>/dev/null || echo '')"
+RED="$(tput setaf 1 2>/dev/null || echo '')"
+CYAN="$(tput setaf 6 2>/dev/null || echo '')"
```

Shellcheck (0.11.0)

[warning] CYAN appears unused. Verify use (or export if used externally).

(SC2034)